peril-settings.json

```diff
@@ -91,16 +105,24 @@
             ]
         },
         "Automattic/simplenote-macos": {
-            "pull_request, pull_request.labeled, pull_request.unlabeled": [
-                "Automattic/peril-settings@org/pr/ios-macos.ts",
-                "Automattic/peril-settings@org/pr/label.ts",
+            "pull_request": [
+                "Automattic/peril-settings@org/pr/ios-macos.ts"
+            ],
+            "pull_request.opened, pull_request.labeled, pull_request.unlabeled, issues.opened, issues.labeled, issues.unlabeled": [
+                "Automattic/peril-settings@org/pr/label.ts"
+            ],
+            "pull_request.opened, pull_request.synchronize": [
                 "Automattic/peril-settings@org/pr/release-notes.ts"
             ]
```

Comment:
I noticed that simplenote-macos doesn't use neither the `milestone.ts` rule – that one might make sense given the SNmacOS app might not be on a regular schedule like our other apps? – but nor the `ios-diff-size.ts` rule. Is that intentional?